### PR TITLE
Org Level Community Health Files

### PR DESCRIPTION
**Pull request by stnguyen90**
These files can serve as defaults for all our repos.

### CODE REVIEW

File: ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: "🐛 Bug Report"
```
[the remark this answers is not in the document]
Comment (stnguyen90) [reply ISSUE_TEMPLATE/bug.yaml:1]: Yup!

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
[the remark this answers is not in the document]
Comment (stnguyen90) [reply CODE_OF_CONDUCT.md:1]: 👍

**Comment by stnguyen90** [comment]
> @stnguyen90 I have a question, though, would it make sense to keep these files as is, or is it better to have a separate directory for them within this repository?

@adityaoberai, I think the issue templates need to be in the `.github/ISSUE_TEMPLATE` folder (which i need to fix). All other supported files may be in the root of the repository, the `.github` folder, or the `docs` folder. Do you have a preference for where to put them?

**Review by brandonroberts — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,465 @@
+# Contributing
```
[the remark this answers is not in the document]
Comment (eldadfux) [reply CONTRIBUTING.md:1]: This file should be deleted or heavily re-written to become a generic contribution guide to Appwrite or our supporting docs. Focusing on code and non-code contributions (tutorials, feedback, issues, discussions, social media, demos, etc...).

I suggest we remove it from here, and tackle it in a dedicated PR.

[the remark this answers is not in the document]
Comment (stnguyen90) [reply CONTRIBUTING.md]: Removed!

**Review by eldadfux — APPROVED** [review]

**Pull request merged**